### PR TITLE
[front] feat: better contrast between text and bg in the FAQ page

### PR DESCRIPTION
**Pull request by GresilleSiffle**
The look and feel of the FAQ page was a bit raw, and its text not very pleasant to read.

In order to ease the reading I added a white `<Paper>` for each question, increasing the contrast between the text and the brackground.

| before                                                                                                             | after                                                                                                             |
|--------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
| capture2 |capture |

As suggested I added a copy URL button. A click a this button displays a visual feedback with `notistack`.

https://user-images.githubusercontent.com/39056254/195399542-54d45c66-8bce-4d91-b435-dbd573db2781.mp4

### CODE REVIEW

File: frontend/src/pages/faq/FAQEntryList.tsx [hunk 1]
```diff
@@ -17,26 +26,63 @@ import { FAQEntry } from 'src/services/openapi';
...
+                <Grid container>
+                  <Grid item xs={11}>
+                    <Typography id={entry.name} variant="h4" gutterBottom>
+                      {entry.question}
+                    </Typography>
+                  </Grid>
+                  <Grid item xs={1}>
+                    <Box
+                      display="flex"
+                      alignItems="center"
```
Comment (lfaucon) [inline frontend/src/pages/faq/FAQEntryList.tsx:59]: Screenshot 2022-10-16 at 20 30 00

The button and question title are not well aligned. Somehow the height of the button forced higher height on the title's Grid container. This also means more blank space before the answer's paragraphs.

(not a high priority to fix and not blocking for merging the PR)

File: frontend/src/utils/notifications.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// The default `autoHideDuration` used by the package notistack is 5000 ms.
+// This value is good for form success or error messages, as it gives the
+// users enough time to read. For more simple operations, like clicking on
+// copy button, a quicker feedback appropriate.
+export const SNACKBAR_DYNAMIC_FEEDBACK_MS = 1800;
```
Comment (lfaucon) [inline frontend/src/utils/notifications.ts:5]: In another open PR, I see camelcase utils constants. Should we have a uniform way for all constants? Are there different types of constants for which the two styles make sense?

**Review by lfaucon — APPROVED** [review]

**Comment by lfaucon** [comment]
Great change! Thanks 👍

File: frontend/src/pages/faq/FAQEntryList.tsx (hunk 1 above)
Comment (GresilleSiffle) [reply frontend/src/pages/faq/FAQEntryList.tsx:59]: It's true, this is due to the button padding. I disabled it in my first draft but I realized removing the padding makes the clickable area of button much more smaller.

I'll try to add a `padding-top` on the title.

File: frontend/src/utils/notifications.ts (hunk 2 above)
Comment (GresilleSiffle) [reply frontend/src/utils/notifications.ts:5]: I'm not sure if there is a consensus in JavaScript, but I know that on the "Google JavaScript Style Guide" constants are uppercase and snake case.

See: https://google.github.io/styleguide/jsguide.html#naming-constant-names

File: frontend/src/pages/faq/FAQEntryList.tsx (hunk 1 above)
Comment (GresilleSiffle) [reply frontend/src/pages/faq/FAQEntryList.tsx:66]: Fixed by https://github.com/tournesol-app/tournesol/pull/1165/commits/a8b731aa52b7c47f801ae80d85c982fd6c76f02f.

**Pull request merged**